streams/src/main/java/org/apache/kafka/streams/state/internals/AbstractDualSchemaRocksDBSegmentedBytesStore.java

```diff
@@ -241,10 +241,8 @@ public String name() {
         return name;
     }
 
-    @Deprecated
-    @Override
-    public void init(final ProcessorContext context,
-                     final StateStore root) {
+    private void initInternal(final ProcessorContext context,
```

Comment:
Could you please merge `initInternal` into `init`? we don't need to have a private method used to generate a small public method :)